www/cordova_index.html

```diff
@@ -14,6 +14,8 @@
   limitations under the License.
 -->
 <!DOCTYPE html>
+<!-- Default LTR direction will be changed dynamically for RTL locales. -->
+<html dir="ltr">
```

Comment:
Does the `dir` attribute have to be on the `html` element? If we can push it down to `app-root` I see two improvements:
* define it just once
* co-locate its definition with where it's set (`app-root.html`)